### PR TITLE
gef.sh is now works on Ubuntu 21.04 (Hirsute Hippo)

### PR DESCRIPTION
**Pull request by BadSugar**
### Description/Motivation/Screenshots ###

Fixing those errors:
* test: unexpected operator
* curl: not found

### How Has This Been Tested? ###

| Architecture |          Yes/No          | Comments                                  |
| ------------ | :----------------------: | ----------------------------------------- |
| x86-32       | :heavy_multiplication_x: | Replace with :heavy_check_mark: if tested |
| x86-64       | :heavy_check_mark: |                                           |
| ARM          | :heavy_multiplication_x: |                                           |
| AARCH64      | :heavy_multiplication_x: |                                           |
| MIPS         | :heavy_multiplication_x: |                                           |
| POWERPC      | :heavy_multiplication_x: |                                           |
| SPARC        | :heavy_multiplication_x: |                                           |
| RISC-V       | :heavy_multiplication_x: |                                           |
| `make test`  | :heavy_multiplication_x: |                                           |

### Checklist ###

- [x] My PR was done against the `dev` branch, not `master`.
- [x] My code follows the code style of this project.
- [x] My change includes a change to the documentation, if required.
- [x] My change adds tests as appropriate.
- [x] I have read and agree to the **CONTRIBUTING** document.

### CODE REVIEW

File: scripts/gef.sh [hunk 1]
```diff
@@ -3,17 +3,22 @@
...
 
 # Backup gdbinit if any
-test -f "${HOME}/.gdbinit" && mv "${HOME}/.gdbinit" "${HOME}/.gdbinit.old"
+if [ -f "${HOME}/.gdbinit" ]; then
+    mv "${HOME}/.gdbinit" "${HOME}/.gdbinit.old"
+fi
 
 # Get the hash of the commit
-ref=$(curl --silent https://api.github.com/repos/hugsy/gef/git/ref/heads/${branch} | grep '"sha"' | tr -s ' ' | cut -d ' ' -f 3 | tr -d "," | tr -d '"')
+ref=$(wget -q -O- https://api.github.com/repos/hugsy/gef/git/ref/heads/${branch} | grep '"sha"' | tr -s ' ' | cut -d ' ' -f 3 | tr -d "," | tr -d '"')
```
Comment (hugsy) [inline scripts/gef.sh:18]: Some distros do not have `wget` by default, `curl` is usually always found and I'm surprised Ubuntu changed that.
Can you change your PR to determine whether `curl` or `wget` exist on the system, and then use the found command to download gef ?

**Review by hugsy — CHANGES_REQUESTED** [review]

File: scripts/gef.sh (hunk 1 above)
Comment (Grazfather) [reply scripts/gef.sh:18]: We could make it so the script takes an arg e.g. `--curl` if you want to use curl or vice versa.

Comment (hugsy) [reply scripts/gef.sh:18]: True, but we still need a default case, and we might as well try to determine it dynamically.

**Review by hugsy — APPROVED** [review]

**Comment by hugsy** [comment]
@BadSugar  Why was the PR closed ? You don't want to see it merged ?

**Pull request closed without merging**